src/submission/submission-updates-messenger.ts

```diff
@@ -20,49 +20,62 @@ class DummyMessenger implements SubmissionUpdatesMessenger {
   };
 }
 
-class KafkaMessenger implements SubmissionUpdatesMessenger {
-  private producer: Producer;
-  private knownTopics: ExpectedTopicsMap;
+const createKafkaMessenger = async (
```

Comment:
I've changed this into a function that returns the object, because I wanted it to run the create topic during messenger object creation and awaits can't be used in constructors

Reply:
Welcome to the dark side :)))

Reply:
Its a good thing we have darkvision :wink: